### ISSUE TITLE
ErrorColumnItem Min/Max Error Values

### ISSUE THREAD

**Issue by pentarex** [oxyplot/oxyplot#823]
So I got stumble to this in my case problem with OxyPlot. I saw that is not possible to set the error to be with different min max values. The method as I see is "public ErrorColumnItem(double value, double error, int categoryIndex = -1)" and If i am correct is for Gaucian plots only. Is there a possibility to add this enhancement in some new release. Because maybe somewhere out there in the world there is another person like me (confused and min/maxless :D) who would like this feature. 

Thanks and if I am missing some point I am sorry.

**Comment by objorke** [comment]
Can this be called asymmetric error bars? See ftp://root.cern.ch/root/doc/ROOTUsersGuideHTML/ch04s04.html
http://se.mathworks.com/matlabcentral/answers/197289-using-errorbar-in-matlab-with-asymmetric-errorbars?requestedDomain=www.mathworks.com

I think it should be possible to do some refactoring to support this, without introducing breaking changes. Create an `AsymmetricErrorColumnItem` class?

**Comment by pentarex** [comment]
Yeah. There is possibility also to do it in excel. It looks something like this http://www.ats.ucla.edu/stat/stata/faq/barcap6.gif (hope its opening the link)
